hera_mc/tests/test_utils.py

```diff
@@ -1,9 +1,20 @@
-import nose.tools as nt
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2017 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+from __future__ import absolute_import, division, print_function
+
+import pytest
+import numpy as np
+import numpy.testing as npt
+
 from astropy.time import Time
 from astropy.units import Quantity
+
 from .. import utils
 
-sidesec = Quantity(1, 'sday').to('day').value  # length of sidereal second in SI seconds.
+# length of sidereal second in SI seconds.
+sidesec = Quantity(1, 'sday').to('day').value
```

Comment:
this variable is only used in one function, I would recommend just moving it into that function.

Reply:
thanks